worker/worker.go

```diff
@@ -114,9 +114,12 @@ type (
 
 		// Start the worker in a non-blocking fashion.
 		Start() error
-		// Run the worker in a blocking fashion. Stop the worker when process is killed with SIGINT or SIGTERM.
+		// Run the worker in a blocking fashion. Stop the worker when ch receives data.
+		// Pass worker.InterruptCh() to stop the worker with SIGINT or SIGTERM.
```

Comment:
The InterruptCh listens explicitly to sigterm only. Does it handle sigterm as well?

Reply:
it is:
```
signal.Notify(c, os.Interrupt, syscall.SIGTERM)
```
and `os.Interrupt` is just an alias to `SIGINT`.